CHANGES.md

```diff
@@ -88,6 +88,8 @@ This change also aligned the authentication architectures between DC/OS Enterpri
 
 ### Fixed and improved
 
+* Metronome fixed failure when restart policy is ON_FAILURE. This bug was introduced through the fix of another bug regarding stopping invalid extra instances of a job run. Metronome should not check the launch queue when a restart is invoked. (DCOS_OSS-4636 )
```

Comment:
I think this is too detailed for dcos changelog. What do you think about just stating something like `Metronome did not handle restart policy is ON_FAILURE correctly, not restarting the task. This is fixed now` or something like that...